components/AppBar/src/MDCAppBarNavigationController.h

```diff
@@ -17,6 +17,7 @@
 @class MDCAppBar;
 @class MDCAppBarViewController;
 @class MDCAppBarNavigationController;
+@class MDCFlexibleHeaderViewController;
```

Comment:
Looks like this is mixing FHV and FHVC in the same PR.  Please split them apart.

Reply:
Obsolete.